.index.json

```diff
@@ -323,6 +323,47 @@
     "crowdsecurity/http-bf-wordpress_bf"
    ]
   },
+  "firewallservices/lemonldap-ng": {
```

Comment:
This should be done by the CI on master, right ?

Reply:
yes, re-running the job to avoid this